storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/JdbcHistoricalRetriever.java

```diff
@@ -326,8 +375,8 @@ String runBatchQuery(
       ResultSet rs = statement.executeQuery(timestampLimitSqlQuery);
 
       while (rs.next()) {
-        Timestamp min_ts = rs.getTimestamp("min"); // Get minimum timestamp
-        Timestamp max_ts = rs.getTimestamp("max"); // Get maximum timestamp
+        Timestamp max_ts = rs.getTimestamp(1); // Get maximum timestamp
```

Comment:
Is this a valid change?

Reply:
Yes, `rs.getTimestamp("max")` doesn't work in Snowflake. So I decided to use the column index number(index starts at 1).

Reply:
Using column index number works in Snowflake and Postgresql

Reply:
Fixed the case sensitive issue. capitalized MAX, MIN  columns in order to fix both snowflake and postgres databases.